contracts/ICSCore.sol

```diff
@@ -88,11 +96,20 @@ contract ICSCore is IICS02Client, IICS04Channel, IICS02ClientErrors, Initializab
 
         emit ICS04ChannelAdded(clientId, channel);
 
+        bytes32 role = _getLightClientRole(clientId);
+        require(_grantRole(role, _msgSender()), Unreachable());
```

Comment:
A couple of product related questions:
* When adding a client, should there be a way to specify a different owner/admin? (maybe not crucial, but might simplify deployment)
* Should we add a way for the owner to transfer ownership? (this could remove the need for the previous question)

Maybe @womensrights can give some thoughts too.

Reply:
We can also create a separate issue for this since there are things to discuss here such as:
- Should you be able to transfer ownership (I think yes)
- Should you be able to specify an admin in creation (I think no)
- Should a seperate admin contract be deployed automatically (this is how proxies do it) where we can also implement transfer and timelock

Reply:
Agree that we can do this as a separate issue 👍
Created an issue and added it to the agenda for the next tech sync call
https://github.com/cosmos/solidity-ibc-eureka/issues/211